src/ToricVarieties/LineBundles/auxilliary.jl

```diff
@@ -0,0 +1,24 @@
+
+function command_string(v::AbstractNormalToricVariety, c::Vector{fmpz})
+    # Initialize a list of strings, which we will eventually join to form the command string
+    string_list = Vector{String}()
+    
+    # cohomCalg cannot handle "[" nor "]" in coordinate names, so we remove those characters.
+    # Note that "[" and "]" appear at other places of the command string.
+    
+    # Add information about Cox ring and its grading to string_list
+    vars = [replace(replace(string(x), "[" => ""), "]" => "") for x in Hecke.gens(cox_ring(v))]
+    divisors = gens(torusinvariant_divisor_group(v))
+    for i in 1:length(vars)
+        push!(string_list, "vertex " * string(vars[i]) * " | GLSM: ( " * join([string(x) for x in map_from_weil_divisors_to_class_group(v)(divisors[i]).coeff], ",") * " )")
```

Comment:
String interpolation, plus not doing too much in one line, makes this more readable, IMHO :-)
```suggestion
        tmp = join([string(x) for x in map_from_weil_divisors_to_class_group(v)(divisors[i]).coeff], ",")
        push!(string_list, "vertex $(vars[i]) | GLSM: ( $(tmp) )")
```

Reply:
Since you use `join([string(x) for x in FOO], ",")` resp. `join([string(x) for x in FOO], ", ")` multiple times (does the `","` vs. `", "` actually matter?), perhaps it'd be useful for readability to also wrap that into an (inner) helper function? Something like
```julia
joincomma(list) = join([string(x) for x in list], ", ")
```

Reply:
I don't think the additional white space matters. (It was there because I found this easier to read. But that is hardly key here.) So I have indeed added this small helper function and removed the white space. The code is not much shorter now, but maybe at least a bit clearer.